cookbooks/aws-parallelcluster-install/recipes/sudoers.rb

```diff
@@ -4,21 +4,26 @@
 # Cookbook:: aws-parallelcluster
 # Recipe:: sudoers
 #
-# Copyright:: 2013-2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# Copyright:: 2013-2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Great catch! Why not 2023?

Reply:
I was considering we were in 2022 :D I still have to adapt my mind.